### DISCUSSION TITLE
Low quality music

### DISCUSSION THREAD

**Discussion by HappyMac3920** [librespot-org/librespot#1118] [Feature Requests]
Are there any plans to include low quality Spotify music in librespot?
I am talking about this (from https://support.spotify.com/us/article/audio-quality/):
"Low: Equivalent to approximately 24kbit/s"
It is on desktop, mobile etc. devices and the file format is still OGG Vorbis.

**Comment by roderickvd** [comment]
It does not scratch my itch so to say, but would be certainly open to accept any PRs for it.
